functionality/unlinkdevice.py

```diff
@@ -0,0 +1,36 @@
+import paho.mqtt.client as mqtt
+import serial
+import register
+import json
+import iddevice
+
+'''
+Unlink Device, by publishing ID to Cloud, and it will find in the dynamoDB database and update by removing it
+'''
+
+broker_address = "110.174.81.168"
+port = 1883
```

Comment:
store IP & port in configuration/ constant file?

Reply:
Modified to load from .env and added bash script to generate .env